javascript/javascript1/week4/voiceassistant.js

```diff
@@ -0,0 +1,82 @@
+  let myName = "";
+  let toDo = [];
+  ////////////Adding name////////////////////
+  function getReply(command) {
+
+    if (command.includes("Hello my name is")) {
+      let strList = command.split(' ')
+
+      if (myName == " ") {
+        return 'Enter your name'
+      }
+      myName = strList[4];
+      return `Nice to meet you ${myName}`
+
+    }
+    if (command.includes("What is my name?")) {
+      return `My name is ${myName}`;
+    }
+
+    /////////////////Today's Date///////////////
+    if (command.includes("What day is it today?")) {
+      return (new Date().toDateString());
+    }
+
+    ////////////////// ToDoList Exercise////////////////
```

Comment:
instead of comments consider extracting functionality into smaller functions with fitting names

Reply:
Created smaller functions